plugins/network/bandwidth

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+#By Jose Manuel Febrer Cortés with Marco Bertola help.
```

Comment:
Please turn the documentation into a perldoc style documentation header.
Maybe see plugins/munin/munin_events for an example.